src/Sets/Zonotope.jl

```diff
@@ -7,6 +7,7 @@ export Zonotope,
        remove_zero_generators
 
 using LazySets.Arrays: _vector_type, _matrix_type
+using StaticArrays: SMatrix, SVector, MMatrix, MVector
```

Comment:
You need to work with `Requires` for optional dependencies. In a nutshell, create a file `init_StaticArrays` [here](https://github.com/JuliaReach/LazySets.jl/tree/master/src/Initialization) and wrap your code in a macro + helper function. An example for `init_Expokit.jl` is [here](https://github.com/JuliaReach/LazySets.jl/blob/a581de8c90d8e3c6d9689a7d724f9c77d7347225/src/LazyOperations/ExponentialMap.jl#L13-L18). It is a bit lengthy to describe this process. We can discuss further in Gitter if you need help.